api/src/pcapi/core/offerers/api.py

```diff
@@ -1893,8 +1894,27 @@ def invite_members(offerer: models.Offerer, emails: list[str], current_user: use
     )  # TODO à modifier envoyer email seulement aux nouveaux invités
 
 
-def get_offerer_members(offerer: models.Offerer) -> list[models.UserOfferer]:
-    return models.UserOfferer.query.filter_by(offererId=offerer.id).join(users_models.User).all()
+def get_offerer_members(offerer: models.Offerer) -> list[typing.Tuple[str, OffererMemberStatus]]:
+    users_offerers = (
+        models.UserOfferer.query.filter(
+            models.UserOfferer.offererId == offerer.id, models.UserOfferer.validationStatus != ValidationStatus.REJECTED
+        )
+        .join(users_models.User)
+        .all()
+    )
+    invited_members = models.OffererInvitation.query.filter_by(offererId=offerer.id).all()
+    members = [
+        (
+            user_offerer_email.user.email,
+            OffererMemberStatus.VALIDATED
+            if user_offerer_email.validationStatus == ValidationStatus.VALIDATED
```

Comment:
Suggestion : de même que plus haut, tu peux utiliser la propriété hybride : 
```suggestion
            if user_offerer_email.isValidated
```

Reply:
Oui j'ai oublié de renommer la variable après un changement de logique.